tfexec/untaint.go

```diff
@@ -0,0 +1,46 @@
+package tfexec
+
+import (
+	"context"
+	"os/exec"
+)
+
+type untaintConfig struct {
+	state string
+}
+
+var defaultUnTaintOptions = untaintConfig{}
+
+// OutputOption represents options used in the Output method.
+type UnTaintOption interface {
+	configureUnTaint(*untaintConfig)
+}
+
+func (opt *StateOption) configureUnTaint(conf *untaintConfig) {
+	conf.state = opt.path
+}
+
+// Untaint represents the terraform untaint subcommand.
+func (tf *Terraform) UnTaint(ctx context.Context, address string, opts ...UnTaintOption) error {
```

Comment:
```suggestion
func (tf *Terraform) Untaint(ctx context.Context, address string, opts ...UnTaintOption) error {
```
Nitpick - I would say that this name would be justified in case the command name was e.g. `un-taint`.
Usually (from my own experience) every 2nd and other capital in `PascalCase` or `camelCase` represents a new word or part of word that is somehow separated from the preceding one.

`untaint` is a whole word though, so I think we can just avoid that extra capital in the middle.